static/styles/rewards/claim-table.css

```diff
@@ -134,6 +134,9 @@ table[data-claim-rendered] #controls {
 table[data-claim-rendered] button {
   opacity: 0.5;
 }
+table[data-claim-rendered] button.hide {
+  display:none !important;
```

Comment:
!important seems like a bad idea. You should use more specific selectors instead, or put this at the bottom of the css file.

Reply:
didn't want to make it more complicated for a simple hide function for this kind of feature, 
otherwise we need to double every selector for buttons. 
if that's okay I'm going to move the `!important` to the bottom of the css file.